test/integration/targets/inventory_aws_ec2/templates/inventory.yml

```
@@ -10,3 +10,4 @@ filters:
 hostnames:
   - tag:Name
   - dns-name
+  - [ "tag:Name", private_ip_address ]
```

```suggestion
  - [ "tag:Name", private-ip-address ]
```
 But this could probably use a new test inventory template with a concatenated hostname option given as a higher precedence instead so it's actually used. There should also be an assert to check that the hostname is correctly concatenated.

---

This should actually be tested.